Jenkinsfile

```diff
@@ -11,14 +11,28 @@ pipeline {
         dockerfile true
     }
 
+    parameters {
+        string(name: "BP_REMOTE", defaultValue: "https://github.com/Pelagicore/software-factory.git")
+        string(name: "BP_BRANCH", defaultValue: "master")
+    }
+
     stages {
         stage('Download') {
             steps {
                 // Delete old files
                 sh 'rm -rf .[^.] .??* *'
+		sh "git init"
+		sh  "git remote add jenkins https://github.com/Pelagicore/software-factory.git"
```

Comment:
Shouldn't this use `params.BP_REMOTE` instead?

Reply:
Thanks for pointing out. Actually I am experimenting with jenkinsfile how it behaves in order to complete my task. once I am done with that I will write a clean code and start pull request.